tests/validation/tests/v3_api/test_encryption_secrets.py

```diff
@@ -0,0 +1,48 @@
+from .common import *  # NOQA
```

Comment:
Could you run flake8 on this file ?

Reply:
Changes have been made to address flake8 errors.